tests/library/status.py

```diff
@@ -0,0 +1,113 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from typing import Collection, Tuple
+
+import requests
+
+from adcm_client.objects import Host, Component, ADCMClient, Cluster
+from adcm_pytest_plugin.docker_utils import ADCM
+
+from tests.library.utils import RequestFailedException, get_json_or_text
+
+POSITIVE_STATUS = 0
+_PATH_TO_ADCM_SECRETS = '/adcm/data/var/secrets.json'
+
+
+class ADCMObjectStatusChanger:
+    """
+    Allows you to "change" status of dummy Components and Hosts.
+
+    If you want to "enable" Cluster or Service,
+        you should set positive status on all hosts and components on it.
+    If you want to "disable" Cluster or Service,
+        you should set negative status at least one of its components or hosts.
+
+    After using status-changing methods you'll need to call `reread` method
+        to be able to get newly set status.
+
+    Consider that status != 16 will be reset to 16 after timeout (300 seconds).
```

Comment:
Status value 16 has a specific meaning?

Reply:
Not totally sure, but I think it's "default" status when no info from statuschecker was received

Reply:
if component wasn't found in db map 16 will be returned
``` go
func (db dbMap2) get(key1 int, key2 int) (Status, bool) {
	val, ok := db[key1][key2]
	if ok {
		return val, true
	}
	return Status{Status: 16}, false
}
```